src/main/java/com/exadel/movie_new/model/Movie.java

```diff
@@ -0,0 +1,114 @@
+package com.exadel.movie_new.model;
+
+public class Movie {
+    private String tagline;
+    private String vote_average;

```

Comment:
Please don't use underscore. In java we use camelCase. In case you need name with underscore for client - you can use annotation @JsonProperty("vote_average"). And every time when you object will be transformed to json, the  variable name in this json will be **vote_average** but not **voteAverage**. It also works in reverse version when you get request from client **vote_average** it will be automatically matched with **voteAverage**.